src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+
+import { CephReleaseNamePipe } from '../../../shared/pipes/ceph-release-name.pipe';
+import { SummaryService } from '../../../shared/services/summary.service';
+
+@Component({
+  selector: 'cd-dashboard-help',
+  templateUrl: './dashboard-help.component.html',
+  styleUrls: ['./dashboard-help.component.scss']
+})
+export class DashboardHelpComponent implements OnInit {
+
+  docsUrl: string;
+
+  constructor(private summaryService: SummaryService,
+              private cephReleaseNamePipe: CephReleaseNamePipe) {}
+
+  ngOnInit() {
+    const subs = this.summaryService.summaryData$.subscribe((summary: any) => {
+      if (summary.version.endsWith('(dev)')) {
```

Comment:
Could you please add a simple test for it? Not because it's complicated just to have it covered by tests.

Reply:
@Devp00l After a new iteration on this code, I just decided to remove it to avoid code that is only used on development.